### ISSUE TITLE
Retinaface quant model prerun multithread graph failed

### ISSUE THREAD

**Issue by nullptr-leo** [OAID/Tengine#1115]
The official retinaface model, quant using the following command:

```
./quant_tool_uint8 -m retinaface.tmfile -i /data/images -o retinaface_quant.tmfile -g 3,416,416 -w 104.007,116.669,122.679 -s 0.017,0.017,0.017
```

When exec on rockchip rv1126, prerun multithread graph failed:

```
tengine-lite library version: 1.4-dev
timvx inited
network input info
 - height: 416
 - width: 416
 - stride: 1248
tengine options
 - num_thread: 1
 - precision: 3
Tengine Fatal: Pre-run subgraph(0) on TIMVX failed.
Tengine: Scheduler(sync) prerun failed.
Prerun multithread graph failed
```

With tengine 1.5 official retinaface timvx example:

```
# ./tm_retinaface_timvx -m retinaface_uint8.tmfile -i face5.jpg -r 1 -t 4
tengine-lite library version: 1.5-dev
Tengine Fatal: Pre-run subgraph(0) on TIMVX failed.
Tengine: Scheduler(sync) prerun failed.
Pre-run graph failed
```

The `landmark_uint8.tmfile` and classification quant model works fine.

| model                             | prerun |
| --------------------------------- | ------ |
| FaceDetect (from `TengineKit`)    | failed |
| yolox_s_uint8                     | failed |
| landmark_uint8                    | ok     |
| mobilenet_quant_tflite            | failed |
| mobilenet_v2_1.0_224_quant_tflite | ok     |
| retinaface_uint8                  | failed |
| ultraface_uint8                   | failed |
| yolo-fastest-1.1_uint8            | failed |
| nanodet_uint8                     | failed |
| yolov5s_uint8                     | failed |
| yolov3-tiny_uint8                 | failed |
| yolov4-tiny_uint8                 | failed |

More details:

```
D [compute_node:375]Instance node[108] "RESHAPE" ...
D [compute_node:375]Instance node[109] "CONV2D" ...
D [compute_node:375]Instance node[110] "CONV2D" ...
D [compute_node:375]Instance node[111] "RESHAPE" ...
D [compute_node:375]Instance node[112] "CONV2D" ...
D [compute_node:375]Instance node[114] "RESHAPE" ...
D [compute_node:398]compute node finish with ret 0
E [vsi_nn_SetupGraph:752]after compute_node
E [vsi_nn_SetupGraph:759]now will vsi_nn_TrySetupCompleteSignalNode
E [vsi_nn_SetupGraph:764]now will vsi_nn_setup_binary_graph_inputs_outputs
E [vsi_nn_SetupGraph:777]vsi_nn_SetupGraph return with ret = 0
E [vsi_nn_VerifyGraph:795]vsi_nn_VerifyGraph return with ret = -1
Tengine Fatal: Pre-run subgraph(0) on TIMVX failed.
Tengine: Scheduler(sync) prerun failed.
Pre-run graph failed
```

Compare `mobilenet_quant_tflite.tmfile` with `mobilenet_v2_1.0_224_quant_tflite.tmfile`, `Softmax` not support?

```
D [setup_node:466]Setup node id[103] uid[104] op[SOFTMAX]
D [validate_op_io_types:167]Validate [SOFTMAX]
D [print_tensor:146]in(0) : id[ 184] vtl[1] const[0] shape[ 52, 104, 2, 1    ] fmt[u8 ] qnt[ASM zp=127, scale=0.040488]
D [print_tensor:146]out(0): id[ 187] vtl[1] const[0] shape[ 52, 104, 2, 1    ] fmt[u8 ] qnt[ASM zp=  0, scale=0.003921]
```

**Comment by nullptr-leo** [comment]
now check the model.

**Comment by nullptr-leo** [comment]
`vsi_nn_VerifyGraph` failed for both `FaceDetect` (from `TengineKit`) and `ultraface`. :(

**Comment by nullptr-leo** [comment]
Compose a simple network w/ and w/o `Softmax`, convert and quant. Then prerun the model on rv1126.
The `ConvNet` w/ `Softmax` prerun failed.

```
./convert_tool -f onnx -m ./convnet.onnx -o ./convnet.tmfile
./quant_tool_uint8 -m ./convnet.tmfile -i ./images -o convnet_uint8.tmfile -g 3,32,32 -a MINMAX -w 0,0,0 -s 0.003922,0.003922,0.003922 -c 0 -t 4 -b 1 -
./tool_tm_check_timvx -m convnet_uint8.tmfile -d [1,3,32,32]
```

```
import torch
import torch.nn as nn

from torch.autograd import Variable

class ConvNet(nn.Module):
    def __init__(self):
        super().__init__()

        # origin
        self.conv = nn.Conv2d(3, 16, 3)
        self.drop = nn.Dropout(0.8, inplace=True)
        self.flat = nn.Flatten()
        self.fc = nn.Linear(14400, 3)
        self.softmax = nn.Softmax(dim=1)

    def forward(self, x):
        x = self.conv(x)
        x = self.flat(x)
        x = torch.reshape(self.fc(x), (1, -1,))
        # x = self.softmax(x)
        return x


if __name__ == '__main__':
    # create net
    net = ConvNet()

    # save as onnx
    x = Variable(torch.randn(1, 3, 32, 32))
    input_names = ['data']
    output_names = ['output']
    torch.onnx.export(net, x, "convnet.onnx", export_params=True, input_names=input_names, output_names=output_names, training=False)
```

**Comment by nullptr-leo** [comment]
With `export VIV_VX_DEBUG_LEVEL=4`:

```
Set drv optimization level = 3
Error: Make sure the environment variable VIVANTE_SDK_DIR is set to the same directory when compiling and executing.
Failed to initialize Kernel "vivante.nn.softmaxlayer2.layer" of Node 0xe0bc0 (status = -1)
```

**Comment by nullptr-leo** [comment]
It turns out the issue is caused by Rockchip mini library libVSC_Lite.so. When libVSC.so used, it works fine.
